packages/odyssey-react-mui/src/labs/AppSwitcher/AppSwitcher.tsx

```diff
@@ -0,0 +1,93 @@
+/*!
+ * Copyright (c) 2024-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import styled from "@emotion/styled";
+import { memo } from "react";
+import {
+  DesignTokens,
+  useOdysseyDesignTokens,
+} from "../../OdysseyDesignTokensContext";
+import { OktaAura } from "./OktaAura";
+import {
+  AppSwitcherApp,
+  AppSwitcherAppIconData,
+  AppSwitcherAppSkeleton,
+} from "./AppSwitcherApp";
+import { TOP_NAV_HEIGHT } from "../TopNav";
+
+const APP_SWITCHER_WIDTH = `${64 / 14}rem`;
```

Comment:
@michaeltamaki-okta Is this value meant to scale with the font size? Or, should it be a static px value?

Reply:
If it should be a rem value, we should probably use one of our spacing tokens here

Reply:
That would be Spacing9 I think. The same as top nav.